.github/workflows/general.yaml

```
@@ -0,0 +1,59 @@
+name: "general"
```

```suggestion
name: "test"
```

Or build, or something like that please - "general" doesn't tell me what this is for in the UI